routes/valueRouter.js

```diff
@@ -1,6 +1,123 @@
-// const express = require('express');
-// const { db } = require('../server/db');
+const express = require('express');
+const { db } = require('../server/db');
 
-// Your code below (delete this line when committing):
+const valueRouter = express.Router();
 
-// const valueRouter = express.Router();
+valueRouter.get('/', async (req, res) => {
+  try {
+    const allValues = await db.query(`
+        SELECT *
+        FROM business;
+      `);
+    res.status(200).send(allValues);
+  } catch (err) {
+    res.status(500).send(err.message);
+  }
+});
+
+valueRouter.get('/:id', async (req, res) => {
+  try {
+    const { id } = req.params;
+    const value = await db.query(
+      `
+        SELECT *
+        FROM donation_tracking
+        WHERE donation_id = $1;  
+      `,
+      [id],
+    );
+    res.status(200).send(value);
+  } catch (err) {
+    res.status(500).send(err.message);
+  }
+});
+
+valueRouter.delete('/:id', async (req, res) => {
+  try {
+    const { id } = req.params;
+    await db.query('DELETE from donation_tracking WHERE donation_id = $1;', [id]);
+    res.status(200).send('Deleted donation');
+  } catch (err) {
+    res.status(500).send(err.message);
+  }
+});
+
+valueRouter.put('/:id', async (req, res) => {
+  try {
+    const { id } = req.params;
+    const {
+      business_id,
+      donation_id,
+      food_bank_donation,
+      reporter,
+      email,
+      date,
+      canned_dog_food_quantity,
+      dry_dog_food_quantity,
+      canned_cat_food_quantity,
+      dry_cat_food_quantity,
```

Comment:
🚫 **[eslint]** <[camelcase](https://eslint.org/docs/latest/rules/camelcase)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Identifier 'dry_cat_food_quantity' is not in camel case.